app/requirements.txt

```diff
@@ -29,13 +29,15 @@ model-mommy==1.6.0
 psycopg2-binary==2.7.7
 pyexcel==0.5.14
 pyexcel-xlsx==0.5.7
+pyjwt>=1.7.1
 python-dateutil==2.7.3
+python-jose>=3.0.0
```

Comment:
Instead of adding the `pyjwt` and `python-jose` dependencies explicitly, consider instead adding the `openidconnect` extra dependency for `social-auth-core` which will include these dependencies (see [requirements-openidconnect.txt](https://github.com/python-social-auth/social-core/blob/9d93069564a60495e0ebd697b33e16fcff14195b/requirements-openidconnect.txt)) and also will ensure that the dependencies stay in tune with any changes that the social auth project makes in the future.